api/python/provisioner/commands/validator/validator.py

```diff
@@ -0,0 +1,250 @@
+import logging
+from abc import abstractmethod
+from pathlib import Path
+from typing import Dict, Callable
+
+from ...errors import ValidationError
+
+logger = logging.getLogger(__name__)
+
+
+class SchemeValidator:
```

Comment:
didn't you forget to inherit form `ABC` here?

Reply:
Yes, it should be inherit from `ABC`. I fixed that.